x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/monitors_page/management/monitor_list_table/monitor_locations.tsx

```diff
@@ -17,22 +17,20 @@ interface Props {
 }
 
 export const MonitorLocations = ({ locations, monitorId, overviewStatus }: Props) => {
-  const {
-    eui: { euiColorVis9, euiColorVis0, euiColorDisabled },
-  } = useTheme();
+  const { euiTheme } = useEuiTheme();
 
   const locationsToDisplay = locations.map((loc) => {
     const locById = `${monitorId}-${loc.id}`;
 
     let status: string = 'unknown';
-    let color = euiColorDisabled;
+    let color = euiTheme.colors.disabled;
 
     if (overviewStatus?.downConfigs[locById]) {
       status = 'down';
-      color = euiColorVis9;
+      color = euiTheme.colors.vis.euiColorVis6;
```

Comment:
This will change colors for Amsterdam. This might need a designer check.
If this is not expected, we'll need to conditionally update.

```
color = isAmsterdam ? euiTheme.colors.vis.euiColorVis9 : euiTheme.colors.vis.euiColorVis6;
```